core/src/main/java/com/ritense/valtimo/service/ChoiceFieldValueService.java

```diff
@@ -113,7 +114,7 @@ public void delete(Long id) {
      */
     @Transactional(readOnly = true)
     public Page<ChoiceFieldValue> findAllByChoiceFieldKeyName(Pageable pageable, String choiceFieldName) {
-        logger.debug("Request to get all ChoiceFieldValues for choiceField {}", choiceFieldName);
+        logger.debug(REQUEST_CHOICEFIELD_VALUES, choiceFieldName);
```

Comment:
Honestly i would change the logged text to reflect what is different. That way you can always find the specfific line in the code, and the information is more accurate
```suggestion
        logger.debug("Request to get all ChoiceFieldValues for choiceField with name '{}'", choiceFieldName);
```